### DISCUSSION TITLE
Does the tally notification work for you?

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#185] [Q&A]
https://notify.tally.xyz/

I subscribe several DAO on Tally including DCDAO and KtonDAO, it seem that the email notification is not working for me, does that work for you?

**Comment by SasoLithops** [comment]
I didn't work for me too. I sent my email again and now subscribed to RING DAO and KTON DAO from here when registering mail. ( https://notify.tally.xyz/home ) Will let you know if it works now...
